packages/react/src/components/DetailsList/DetailsRow.types.ts

```diff
@@ -275,6 +275,9 @@ export type IDetailsRowStyleProps = Required<Pick<IDetailsRowProps, 'theme'>> &
 
   /** Whether to animate updates */
   enableUpdateAnimations?: boolean;
+
+  /** Defines the selection mode of list among single, multi or none */
```

Comment:
The `DetailsRow` component actually already has access to `selection.mode` from the `Selection` object itself.